test/parallel/test-event-emitter-add-listeners.js

```diff
@@ -42,7 +42,7 @@ e.emit('hello', 'a', 'b');
 
 
 // just make sure that this doesn't throw:
-var f = new events.EventEmitter();
+var f = new EventEmitter();
```

Comment:
prefer wrapping in a block rather than having a new variable name?

or is that too large of a change?